admin/tests/acceptance/authenticated/users/get-test.js

```diff
@@ -32,6 +32,7 @@ module('Acceptance | authenticated/users/get', function (hooks) {
     const expectedParticipationCount = 1;
     const expectedCertificationCenterCount = 3;
     const expectedAuthenticationMethodCount = 3;
+    const connections = this.intl.t('pages.user-details.navbar.connections');
```

Comment:
Une suggestion pour un nommage qui serait un peu plus indicatif :
```suggestion
    const connectionTabLabel = this.intl.t('pages.user-details.navbar.connections');
```